app/views/filters/_form.html.erb

```diff
@@ -48,7 +48,8 @@
 
         <%= autocomplete_f f, :search,
                            :disabled => f.object.unlimited?,
-                           :path => resource_path(f.object.resource_type) || "",
+                           :path => "",
```

Comment:
I'm slightly confused, I thought on your previous update that you'd fixed this, and there was a conditional checking for full_path in autocomplete_f?